Lib/test/test_clinic.py

```diff
@@ -3288,6 +3288,14 @@ def test_cloned_func_with_converter_exception_message(self):
                 func = getattr(ac_tester, name)
                 self.assertEqual(func(), name)
 
+    def test_meth_method_no_params(self):
+        obj = ac_tester.TestClass()
+        meth = obj.meth_method_no_params
+        check = partial(self.assertRaisesRegex, TypeError, "no arguments")
+        check(meth, 1)
+        check(meth, a=1)
+        obj.test_meth_method_no_params_empty_kw_tuple()
```

Comment:
I think that it would be simpler to write this test in Python, using `_testcapi.pyobject_vectorcall()`.

Reply:
Oh, how convenient! TIL :) Addressed in 088a3b80f4a779521b8e48e2c318c9c247e0b7cc